### ISSUE TITLE
Appropriate analysis of closely eluted isobaric species 

### ISSUE THREAD

**Issue by QuiqMan** [mzmine/mzmine2#828]
### Basic information

* My operating system: Windows 10 pro 64 bits
* My MZmine version:  2.53

### What happened

Dear MZmine crew, 

I am having problems to appropritately deternime closely eluted isoleucine/leucine isobaric species ([M+H]+ 132.1018 at Rt 2.56 and 2.75 min). The LC-MS system was a Q-Exactive Orbitrap. It was expected to find two peaks merging leucine/isoleucine tentative identifications at such aforementioned retention times with correct integrated peak areas. Peak detection and integration was manually performed by Thermo Xcalibur for quality assessment of MZmine results. Considered mass tolerance was 5 ppm. Since the analysis had exploratory purposes, retention time tolerance was set according to the running time (30 min) in first two approaches performed.

Three strategies were followed:

1- Raw data impot-->Mass detection-->Targeted feature detection (EXPLORATORY, Rt tolerance was 30 min))

Expeceted results (including integrated peak area) were achieved at Rt 2.75 min but nothing appeared at 2.56 min.

2- Raw data import-->Mass detection-->ADAP chromatogram builder (EXPLORATORY, Rt tolerance was 30 min

There was found only one peak merging integrated peak areas. Resultant integrated area was right considering summation of areas from individual peaks. Furhter analysis using Deconvolution, Isotopic peak grouper and Identfication modules gave isoleucine as an isolated result.

3- Raw data impot-->Mass detection-->Targeted feature detection (TARGETED, Rt tolerance was 0.1 min). Chromatograhic retention time for isoleucine and leucine were from previous studies.

Correct identifications and integrated peak areas were achieved.

In summary, under constrained targeted analysis conditions in where the retention time is 100% critical the analysis worked fine. However, in exploratory analyses in where the Rt lacks of sense the results were not satisfactory.  

[Isoleucine_leucine_MZmineScreenshots.pdf](https://github.com/mzmine/mzmine2/files/4614898/Isoleucine_leucine_MZmineScreenshots.pdf)

**Comment by tomas-pluskal** [comment]
Hi. I don't really understand what your question is, could you state your question?
The results you obtain depend strongly on ALL parameter values, not just on Rt tolerance.

**Comment by QuiqMan** [comment]
Hi there!!. My question was about why in an untargeted analysis (Rt
dependence OFF) I did not get two separated isobaric peaks appropriately
integrated instead of what I got that was just one?. Peak picking in the
first approach only gave me the appropriate integration of the second peak
at Rt 2.75 min (but nothing about the first one at 2.55 min) whereas in the
second approach, integration of the only peak detected was the summation of
areas from peaks 1 and 2. I did not have any problem in exploratory
analyses with non-isobaric species even with same Rt.

Many thanks and sorry for any disturb,

QuiqMan

**Comment by tomas-pluskal** [comment]
Please upload your .mzmine project file that demonstrates the issue so we can reproduce it. And list the exact parameter values that you used for each step of the analysis.

**Comment by QuiqMan** [comment]
[isoleucine_leucine_MZmineproject.zip](https://github.com/mzmine/mzmine2/files/4621298/isoleucine_leucine_MZmineproject.zip)

Hope this can help.

Cheers!!

**Comment by tomas-pluskal** [comment]
The fact that the ADAP Chromatogram Builder detects both peaks in a single chromatogram is *correct*. You need to use the deconvolution module properly to separate the peaks. Why did you choose the Savitzky-Golay algorithm? Did you try the Preview function for the deconvolution step?

Try using the Local minima search instead with the following parameters, that can separate your peaks quite nicely:
<img width="1298" alt="Screen Shot 2020-05-13 at 23 52 33" src="https://user-images.githubusercontent.com/1796692/81828704-0c66d700-9575-11ea-81ac-e00468f8e60d.png">

**Comment by QuiqMan** [comment]
Awesome!!, tested what you suggested with the Local minimum deconvolution and performance achieved was perfect. 

Thank you so much for your attention and super fast responses and guidance. 

Cheers!!
